doc/source/guide/data_types/maps.rst

```diff
@@ -45,10 +45,10 @@ To do this you need to provide `~sunpy.map.map_factory.MapFactory` with both the
 well as some basic meta information. If no header is given then some default
 values as assumed. Here is a simple example::
 
-    import numpy as np
-    data = np.arange(0,100).reshape(10,10)
-    header = {'cdelt1': 10, 'cdelt2': 10, 'telescop':'sunpy'}
-    my_map = sunpy.map.Map(data, header)
+    >>> import numpy as np
+    >>> data = np.arange(0,100).reshape(10,10)
+    >>> header = {'cdelt1': 10, 'cdelt2': 10, 'telescope':'sunpy'}
```

Comment:
Sorry, I didn't realized this. Fixed now (and didn't use [skip ci] to have a chance that it actually runs this time on travis).